dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -299,6 +299,8 @@ def wait_for_element(selection_criteria, enabled)
   wait_until do
     element = @browser.find_element(selection_criteria)
     element.enabled? == enabled
+  rescue Selenium::WebDriver::Error::StaleElementReferenceError
```

Comment:
@stephenliang , I added you to get your inputs specifically on this change. I noticed that the test sometimes failed when checking for enabled with stale element reference exception and this was to fix the same.

Reply:
I don't think this is a test bug but rather a true bug that happens for our users and the stale element reference is legitimate.

The selenium video shows that after pressing edit all, the family name input box reverts to non-editing state. I can confirm that I am able to reproduce this as a normal user and is due to a race condition. In Redux, we set `manageStudents/SET_STUDENTS` twice, but if `manageStudents/EDIT_ALL` happens between the first and second set students, it will turn on and then off edit mode.

So I would say that this is a product bug - the test is flaky because there's a bug in the product. Attached a video where I reproduced it twice.

This would likely result in the test retrying and possibly passing? It seems if the timing is just right it will pass but I don't think the changes being proposed in this PR is the right way to fix this bug, instead we should fix the race condition.

Reply:
PR to address the product issue https://github.com/code-dot-org/code-dot-org/pull/60972

I'll hold on merging this PR until the product fix is place.